src/components/Reports/GoalCalculator/CalculatorSettings/Steps/InformationStep/InformationStepForm/InformationStepFinancialForm.tsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 import { Grid, MenuItem, TextField, Typography } from '@mui/material';
+import { styled } from '@mui/system';
 import { Field, FormikProps } from 'formik';
 import { useTranslation } from 'react-i18next';
 
+const StyledFinancialForm = styled('div')(() => ({
```

Comment:
When a styled component doesn't need the theme, the styles don't have to be in an arrow function.

```suggestion
const StyledFinancialForm = styled('div')(({
```